pkg/transaction/definitions.go

```diff
@@ -62,16 +62,16 @@ type LogEvent struct {
 
 // Summary - Represent the transaction summary event
 type Summary struct {
-	Status       string       `json:"status,omitempty"`
-	StatusDetail string       `json:"statusDetail,omitempty"`
-	Duration     int          `json:"duration"`
-	Application  *Application `json:"application,omitempty"`
-	Product      *Product     `json:"product,omitempty"`
-	Team         *Team        `json:"team,omitempty"`
-
-	Proxy      *Proxy      `json:"proxy,omitempty"`
-	Runtime    *Runtime    `json:"runtime,omitempty"`
-	EntryPoint *EntryPoint `json:"entryPoint,omitempty"`
+	Status                       string       `json:"status,omitempty"`
+	StatusDetail                 string       `json:"statusDetail,omitempty"`
+	Duration                     int          `json:"duration"`
+	Application                  *Application `json:"application,omitempty"`
+	Product                      *Product     `json:"product,omitempty"`
+	Team                         *Team        `json:"team,omitempty"`
+	Proxy                        *Proxy       `json:"proxy,omitempty"`
+	Runtime                      *Runtime     `json:"runtime,omitempty"`
+	EntryPoint                   *EntryPoint  `json:"entryPoint,omitempty"`
+	IsIncludedInUsageMetricEvent bool         `json:"is_included_in_usage_metric_event"`
```

Comment:
Can we have a short name for the attribute?

Reply:
Spent quite some time on this variable name 😄 how about just _IsIncludedInUsageMetric_?

Reply:
isInUsageMetric?

Reply:
Not concerned about the variable name but the attribute name "is_included_in_usage_metric_event". Can we use "hasMetricEvent" or something similar instead?

Reply:
My idea was that if someone looks at a transaction summary event they can immediately figure out what this attribute exactly means. What if we use _IsInUsageMetric_ as variable name and _is_in_usage_metric_ as attribute name?

Reply:
Changed it to _IsInUsageMetric_ & _is_in_usage_metric_ respectively

Reply:
Can we make the attribute name as well in camel case (isInUsageMetric instead of is_in_usage_metric) to be consistent?

Reply:
I think the final change is already in there as isInUsageMetric for both names